### DISCUSSION TITLE
Languages doesn't show properly

### DISCUSSION THREAD

**Discussion by meppu** [community/community#50448] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

So i have a [repository](https://github.com/cart96/tokyo) which is mostly based on rust and elixir. But github only shows elixir part, what is wrong? Since rust code is most important part of the repository and if i keep add codes to python client, people may think it is written in python.

**Comment by airtower-luna** [comment, marked as answer]
I assume the problem is that the directory that holds the Rust code is called `cache`. GitHub uses Linguist to detect languages, and [as its troubleshooting documentation describes](https://github.com/github/linguist/blob/master/docs/troubleshooting.md) by default it ignores typical paths for vendored (euphemism for "copied from upstream into your repository") code. And literally the [first pattern in the file defining those exclude patterns](https://github.com/github/linguist/blob/c34f887f48d81e2ed42b59825fa5fe12d3918d63/lib/linguist/vendor.yml#L13) will exclude `cache`.

Assuming the directory is named `cache` for a good reason, try looking at the overrides described in the linked docs.

**Reply by meppu** [reply]
Yeah that was the reason, I'll override it. thanks!
